### DISCUSSION TITLE
ZOAU zIIP eligibility

### DISCUSSION THREAD

**Discussion by roded** [IBM/zoau-samples#81] [Ideas]
Hello,
Taking a shot at a possibly naive question:
We're using ansible-collections/ibm_zos_core with zoau under the hood for increasingly large portions of our z/OS CI workload.
While Python, in which ibm_zos_core written, is zIIP eligible, we suspect that zoau's modules are not (our ansible playbooks which utilize zoau considerably seem to mostly run on GCP). 

We were wondering if there are any plans to make the zoau modules zIIP eligible. Such a change would considerably facilitate development on Z for us and I suspect others as well.

Cheers.

**Comment by AnthonyGiorgio** [comment]
The native Python portions of ZOAU are zIIP-eligible, but the native code parts are not.  ZOAU will continue to be a mix of both Python and native code going into the future.

**Reply by roded** [reply]
Thanks for the reply @AnthonyGiorgio.
Is it out of the question for IBM to enable zIIP eligibility for the native parts of ZOAU?

**Comment by AnthonyGiorgio** [comment]
You would have to discuss that with the IBM Python team.  ZOAU has no ability to decide whether code is zIIP eligible or not in Python.

**Reply by roded** [reply]
https://community.ibm.com/community/user/ibmz-and-linuxone/discussion/zoau-ziip-eligibility
